include/private/firebird/common.h

```diff
@@ -16,6 +16,7 @@
 #include <limits>
 #include <sstream>
 #include <iomanip>
+#include <stdint.h>
```

Comment:
It would be more consistent to include `<cstdint>` (here and elsewhere).

Reply:
`<cstdint>` is a C++11 header, which is why I used `<stdint.h>`. But as you're working on dropping C++98 support in #996 , I'll change that of course.